### PR TITLE
[ECS] Refactor sniffs to not show extra errors

### PR DESCRIPTION
**Pull request by alexndlm**
- it is no sense to add text error if we provide fixable error

### CODE REVIEW

File: tests/Sniffs/Namespaces/Psr4Sniff/Fixture/Wrong/NamespaceWithSingleWorld.php.inc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Wrong;
+
+class NamespaceWithSingleWorld
```
Comment (ERuban) [inline tests/Sniffs/Namespaces/Psr4Sniff/Fixture/Wrong/NamespaceWithSingleWorld.php.inc:5]: `NamespaceWithSingleWorld` => `SingleWordNamespace` looks better, imo

File: tests/Sniffs/Namespaces/Psr4Sniff/Psr4SniffTest.php [hunk 2]
```diff
@@ -33,6 +33,16 @@ public function provideFixtures(): iterable
                 ],
             ],
         ];
+
+        yield 'Wrong, namespace with single world' => [
```
Comment (ERuban) [inline tests/Sniffs/Namespaces/Psr4Sniff/Psr4SniffTest.php:37]: `word`

**Review by ERuban — CHANGES_REQUESTED** [review]

**Comment by ERuban** [comment]
Don't we need to fix `AlphabeticallySortedArrayKeysSniff` too?

File: tests/Sniffs/Namespaces/Psr4Sniff/Fixture/Wrong/NamespaceWithSingleWorld.php.inc (hunk 1 above)
Comment (alexndlm) [reply tests/Sniffs/Namespaces/Psr4Sniff/Fixture/Wrong/NamespaceWithSingleWorld.php.inc]: Thanks. Fixed.

File: tests/Sniffs/Namespaces/Psr4Sniff/Psr4SniffTest.php (hunk 2 above)
Comment (alexndlm) [reply tests/Sniffs/Namespaces/Psr4Sniff/Psr4SniffTest.php]: Thanks. Fixed.

**Comment by alexndlm** [comment]
> Don't we need to fix `AlphabeticallySortedArrayKeysSniff` too?

Thanks. Great point. Fixed.

**Review by ERuban — APPROVED** [review]

File: src/Sniffs/Attributes/SortedApiResourceOperationKeysSniff.php [hunk 3]
```diff
@@ -241,12 +241,6 @@ private function processArrayContent(File $phpcsFile, int $bracketOpenerPointer,
             return;
         }
 
-        $phpcsFile->addErrorOnLine(
```
Comment (ERuban) [inline src/Sniffs/Attributes/SortedApiResourceOperationKeysSniff.php:244]: Not related to that PR - please, add the description that this sniff applicable to `api-platform` usage only.
Thank you 🙏

Comment (ERuban) [reply src/Sniffs/Attributes/SortedApiResourceOperationKeysSniff.php:244]: Actually we can get rid of this then, bcs it is actual only for >=2.7 versions of the api-platform

Comment (alexndlm) [reply src/Sniffs/Attributes/SortedApiResourceOperationKeysSniff.php:244]: Done

**Review by roman-eonx — APPROVED** [review]

**Pull request merged**